week3/homework/exercises/4-handlebars/script.js

```diff
@@ -42,4 +31,43 @@ const punchlines = [
   'go to the moon',
   'achieve world piece',
   'help people learn programing',
-];
\ No newline at end of file
+];
+
+/**
+ * Given an array, return an element from it chosen at random
+ */
+function getRandomElement(array) {
+  const randomElement = array[Math.floor(Math.random() * array.length)];
+  return randomElement;
```

Comment:
```suggestion
  return array[Math.floor(Math.random() * array.length)];
```
You can make this a bit shorter 😄